pkg/apis/core/validation/validation_test.go

```diff
@@ -7699,7 +7699,7 @@ func TestValidatePullPolicy(t *testing.T) {
 func TestValidateResizePolicy(t *testing.T) {
 	featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.InPlacePodVerticalScaling, true)
 	tSupportedResizeResources := sets.NewString(string(core.ResourceCPU), string(core.ResourceMemory))
-	tSupportedResizePolicies := sets.NewString(string(core.NotRequired), string(core.RestartContainer))
+	tSupportedResizePolicies := sets.NewString(string(core.ResizeRestartPolicyPreferNoRestart), string(core.ResizeRestartPolicyNotRequired), string(core.ResizeRestartPolicyRestartContainer))
```

Comment:
I would expect to see a bunch of duplicated test cases if we have two ways of expressing the same thing

Reply:
Done, but programmatically rather than manually duplicating each.